06-building-a-program-start-to-finish/writing-tests/output/InvertedIndex.spec.js

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable no-undef */
+const invertedIndex = new InvertedIndex();
+const file1 = require('../books/adventureBooks.json');
+const file2 = require('../books/emptyBook.json');
+const file3 = require('../books/wrongFormat.json');
+
```

Comment:
Is this proper naming conventions? I believe you can use names that gives an understanding of what these files are for. file1, file2, file3 doesn't communicate properly.